src/main/java/it/pagopa/ecommerce/payment/methods/controller/PaymentMethodsController.java

```diff
@@ -289,4 +292,16 @@ private Mono<String> getAuthenticationToken(ServerWebExchange exchange) {
                         .map(header -> header.substring("Bearer ".length()))
         );
     }
+
+    @Warmup
```

Comment:
What about adding also a method to perform warmup for a calculate fees endpoint too? 
This should be a safe operation to be performed as warmup requests.

cc @infantesimone

Reply:
done